src/pages/iou/steps/IOUConfirmPage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from '../../../ONYXKEYS';
+import styles from '../../../styles/styles';
+import ButtonWithLoader from '../../../components/ButtonWithLoader';
+
+const propTypes = {
+    // Callback to inform parent modal of success
+    onConfirm: PropTypes.func.isRequired,
+
+    // IOU amount
+    iouAmount: PropTypes.number.isRequired,
+
+    /* Onyx Props */
+
+    // Holds data related to IOU view state, rather than the underlying IOU data.
+    appStateIOU: PropTypes.shape({
+
+        // Whether or not the IOU step is loading (creating the IOU Report)
+        loading: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    appStateIOU: {},
+};
+
+const IOUConfirmPage = props => (
+    <View style={styles.settingsWrapper}>
+        <ButtonWithLoader
+            style={[styles.button, styles.w100]}
+            text={`Request $${props.iouAmount}`}
```

Comment:
Typo `$$`?

Reply:
This is just a placeholder, but it's to render the dollar currency symbol: `Request $42.00`